js/libs/keycloak-admin-client/test/timeout.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { expect } from "chai";
+import { KeycloakAdminClient } from "../src/client.js";
+import { credentials } from "./constants.js";
+import { Server, createServer } from "http";
+
+describe("Timeout", () => {
+  let server: Server;
+
+  before(async () => {
+    server = createServer((req, res) => {
+      res.writeHead(200, { "Content-Type": "text/plain" });
+      setTimeout(() => res.end("Hello, world!\n"), 1500);
+    });
+    server.listen(8888, "localhost");
+  });
+
+  after(async () => {
+    server.close();
+  });
+
+  void it("create without timeout", async () => {
+    const client = new KeycloakAdminClient({
+      baseUrl: "http://localhost:8888",
+    });
+
+    try {
+      await client.auth(credentials);
+    } catch (error) {
+      expect(error).to.be.an("Error");
+      expect((error as Error).message).to.contain("Unexpected token 'H'");
```

Comment:
This could likely be a non-error case if the server returned JSON, but I guess this also works for proving the point of the test.